src/components/color-description/color-description.scss

```diff
@@ -1,5 +1,6 @@
 .vibe-sb-comps-color-description {
   border-radius: var(--sb-border-radius-extra-small);
+  width: auto;
```

Comment:
For me it seems to work fine even without this chane

Reply:
Ok will remove it then👍